src/java.desktop/share/classes/com/sun/imageio/plugins/jpeg/JPEG.java

```diff
@@ -199,39 +198,11 @@
 
     static final int [] bOffsRGB = { 2, 1, 0 };
 
-    /* These are kept in the inner class to avoid static initialization
-     * of the CMM class until someone actually needs it.
-     * (e.g. do not init CMM on the request for jpeg mime types)
-     */
```

Comment:
It is not necessary to delay the usage of ColorSpace, it itself defers the usage of cmm.